packages/phone-number-privacy/combiner/test/unit/domain-threshold-state.test.ts

```diff
@@ -26,6 +26,8 @@ describe('domain threshold state', () => {
     const mockRequest = {
       body: {},
     } as Request
+
+    // @ts-ignore: missing some properties
     const mockResponse = {
```

Comment:
I think here the `satisfies` keyword would work https://tusharf5.com/posts/typescript-satisfies-operator/#:~:text=With%20the%20new%20satisfies%20operator,code%3B%20not%20the%20TypeScript%20compiler.

Reply:
satisfies does not work here. I thinks the types might be a too different and causing conflicts. I elected to use `ts-ignore` to limit my changes to what's only required for node18 to work.